CMakeLists.txt

```
@@ -9,6 +9,10 @@ set(CMAKE_CXX_FLAGS "-O3 -march=native")
 # Get includes, which declares the `spblas` library
 add_subdirectory(include)
 
+if (${CMAKE_VERSION} VERSION_GREATER 3.23.99)
```

```suggestion
if (${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.24)
```
it's just my preference. no strong opinion

---

I'm reading the docs just now and see that `VERSION_GREATER` is in all version 3.x CMake release and `VERSION_GREATER_EQUAL` was introduced in version 3.7. So my preference would be to keep compatibility with all  versions. Of course, any version older than 3.20 will be automatically tripped up by the first line asking for CMake 3.20. As you say, it's a matter of preference. It may be better to keep all syntax clean, precise, and up to date. I'm happy to push a quick fix. Just let me know.

---

I pushed 794bcbd to get CMake 3.24 more explicit in comparison, as suggested.

---

I don't have an opinion here---I'm fine with whatever you guys think is best.